robot/Cumulus/resources/RecurringDonationsPageObject.py

```diff
@@ -15,6 +15,13 @@
 class RDListingPage(BaseNPSPPage, ListingPage):
     object_name = "npe03__Recurring_Donation__c"
 
+    @capture_screenshot_on_error
+    def wait_for_rd2_modal(self):
+        """Based on the button name (Cancel)  or (Save) on the modal footer, selects and clicks on the respective button"""
+        btnlocator = npsp_lex_locators["button-with-text"].format("Save")
+        self.selenium.scroll_element_into_view(btnlocator)
```

Comment:
I dont think u would need to scroll to the element in order to wait for it to be visible